bin/deploy-dockerhub.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# THIS IS MEANT TO BE RUN BY CI ONLY.
+
+set -e
+
+# Usage: retry MAX CMD...
+# Retry CMD up to MAX times. If it fails MAX times, returns failure.
+# Example: retry 3 docker push "mozilla/kinto-dist:$TAG"
+function retry() {
+    max=$1
+    shift
+    count=1
+    until "$@"; do
+        count=$((count + 1))
+        if [[ $count -gt $max ]]; then
+            return 1
+        fi
+        echo "$count / $max"
+    done
+    return 0
+}
+
+# configure docker creds
+retry 3  echo "$DOCKER_PASSWORD" | docker login -u="$DOCKER_USERNAME" --password-stdin
+
+# docker tag and push git branch to dockerhub
+if [ -n "$1" ]; then
+    [ "$1" == master ] && TAG=latest || TAG="$1"
+    docker tag kinto-dist "mozilla/kinto-dist:$TAG" ||
+        (echo "Couldn't tag kinto-dist as mozilla/kinto-dist:$TAG" && false)
+    retry 3 docker push "mozilla/kinto-dist:$TAG" ||
+        (echo "Couldn't push mozilla/kinto-dist:$TAG" && false)
+    echo "Pushed mozilla/kinto-dist:$TAG"
```

Comment:
Before we were using the `$DOCKERHUB_REPO` variable

Reply:
Fixed.